monkey/infection_monkey/utils/linux/hidden_files.py

```diff
@@ -0,0 +1,34 @@
+HIDDEN_FILE = '/var/tmp/.monkey-hidden-file'
+HIDDEN_FOLDER = '/var/tmp/.monkey-hidden-folder'
+
+
+def get_linux_commands_to_hide_files():
+    return [
+        'touch',    # create file
+        HIDDEN_FILE,
+        '; echo \"Successfully created hidden file: {}\" |'.format(HIDDEN_FILE),  # output
```

Comment:
`;` is a shell separator which means "run anyway, even if the previous command failed". [Read more here](https://stackoverflow.com/questions/6152659/bash-sh-difference-between-and). I think that `&&` is more fitting here.

Also, instead of piping and `tee`-ing, I think you can just `echo Successfully created hidden file > HIDDEN_FILE`

Reply:
Alright, got it!

I used `tee` because directly `echo`-ing it into the file gives no output, so it shows the default `(PBA execution produced no output)` in the report (which is also seen in the screenshot I attached in the previous comment).

Reply:
Cool, good to know :)